deps/http_parser/http_parser.gyp

```diff
@@ -51,13 +51,12 @@
     {
       'target_name': 'http_parser',
       'type': 'static_library',
-      'include_dirs': [ '.' ],
+      'include_dirs': [ '.', 'src' ],
```

Comment:
since there's already discussion about the format of the source tree. I'd prefer having an explicit `include` directory. (and have `http_parser.h` in it) and `include/http_parser.h` explicitly mentioned in `sources`.
`make` will be much more predictable that way.